client/src/components/Home/index.tsx

```diff
@@ -391,6 +407,8 @@ export default class Home extends React.Component<any, IHomeState> {
             defaultValue={template && template.id || ''}
             lineDirection="center"
             placeholder="Choose an ID for the dashboard (will be used in the url)"
+            error={error}
```

Comment:
I much prefer the contextual inline input errors: 
![screen shot 2017-08-14 at 17 32 24](https://user-images.githubusercontent.com/1880480/29281343-95aefbda-8116-11e7-9473-41af3cae6d39.png) 
I'll have another look at this...